semantic_conventions/resource/cloud.yaml

```
@@ -23,20 +23,26 @@ groups:
       - id: account.id
         type: string
         brief: >
-          The cloud account ID used to identify different entities.
-        examples: ['opentelemetry']
+          The cloud account ID the resource is created in.
```

```suggestion
          The cloud account ID the resource is assigned to.
```

If a cloud provider allows moving resources between regions, I assume we don't want the original region.